plugin/pkg/admission/noderestriction/admission.go

```diff
@@ -125,6 +130,12 @@ func (c *nodePlugin) Admit(a admission.Attributes) error {
 			return admission.NewForbidden(a, fmt.Errorf("may only update PVC status"))
 		}
 
+	case svcacctResource:
+		if c.features.Enabled(features.TokenRequest) {
```

Comment:
I'd actually deny altogether if the feature was disabled

Reply:
Why not leave the current behavior if the feature is turned off? At this point we haven't even checked that the request was to the '/token' subresource. It's a bit weird to be doing denys in an admission controller on the resource attribute alone. Where there is enough information available to the authorizer to deny a request, we should prefer to deny the request there. All the information we would be making this deny decision on is available to the Node authorizer.

Reply:
> Why not leave the current behavior if the feature is turned off?

Agreed. Feature gates should maintain the current behavior when disabled.

Reply:
ok